contracts/network/BancorNetwork.sol

```diff
@@ -216,6 +224,21 @@ contract BancorNetwork is IBancorNetwork, Upgradeable, ReentrancyGuardUpgradeabl
      */
     event NetworkFeesWithdrawn(address indexed caller, address indexed recipient, uint256 amount);
 
+    /**
+     * @dev triggered when pool surplus tokens are withdrawn
+     */
+    event SurplusTokensWithdrawn(
```

Comment:
POLWithdrawn? I think it's clearer to use the POL term everywhere

Reply:
Changed to POL (for function as well)